react/features/base/conference/actions.js

```diff
@@ -154,6 +156,9 @@ function _addConferenceListeners(conference, dispatch) {
     conference.on(
         JitsiConferenceEvents.USER_ROLE_CHANGED,
         (...args) => dispatch(participantRoleChanged(...args)));
+    conference.on(
```

Comment:
This is a great point. Upon further inspection, however, this entire function is not used on web, because initializing the conference is not unified yet, so I consider that out of the scope of this PR.